README.md

```diff
@@ -44,38 +50,52 @@ command line like this:
+
 Normalized, in this case, means:
 
-* The entire CSV is in the UTF-8 character set.
-* The `Timestamp` column should be formatted in RFC3339 format.
-* The `Timestamp` column should be assumed to be in US/Pacific time;
+- The `Timestamp` field should be formatted in RFC3339 format.
+- The `Timestamp` field should be assumed to be in US/Pacific time;
   please convert it to US/Eastern.
-* All `ZIP` codes should be formatted as 5 digits. If there are less
+- All `ZIP` codes should be formatted as 5 digits. If there are less
   than 5 digits, assume 0 as the prefix.
-* The `FullName` column should be converted to uppercase. There will be
+- The `FullName` field should be converted to uppercase. There will be
   non-English names.
-* The `Address` column should be passed through as is, except for
-  Unicode validation. Please note there are commas in the Address
-  field; your CSV parsing will need to take that into account. Commas
-  will only be present inside a quoted string.
-* The `FooDuration` and `BarDuration` columns are in HH:MM:SS.MS
+- The `FooDuration` and `BarDuration` fields are in HH:MM:SS.MS
   format (where MS is milliseconds); please convert them to the
   total number of seconds expressed in floating point format.
   You should not round the result.
-* The `TotalDuration` column is filled with garbage data. For each
+- The `TotalDuration` field is filled with garbage data. For each
   row, please replace the value of `TotalDuration` with the sum of
   `FooDuration` and `BarDuration`.
-* The `Notes` column is free form text input by end-users; please do
-  not perform any transformations on this column. If there are invalid
-  UTF-8 characters, please replace them with the Unicode Replacement
-  Character.
+- The `Notes` field is free form text input by end-users; please do
+  not perform any transformations on this field.
 
-You can assume that the input document is in UTF-8 and that any times
-that are missing timezone information are in US/Pacific. If a
+You can assume that the sample data we provide will contain all date
+and time format variants you will need to handle. Any times that are
+missing timezone information are in US/Pacific
+
+#### For CSV solutions only
+
+- The entire data set is in the UTF-8 character set.
+- The `Address` field should be passed through as is, except for
+  Unicode validation. Please note there are commas in the Address
+  field; your CSV parsing will need to take that into account. Commas
+  will only be present inside a quoted string.
+- If there are invalid UTF-8 characters in the `Notes` field, please
+  replace them with the Unicode Replacement Character.
+
+You can assume that the input document is in UTF-8. If a
 character is invalid, please replace it with the Unicode Replacement
 Character. If that replacement makes data invalid (for example,
 because it turns a date field into something unparseable), print a
 warning to `stderr` and drop the row from your output.
```

Comment:
> > What if we include a screenshot of how the data might be displayed on a webpage? If we can do that in a way that makes clear the candidate doesn't have to follow that exact format but instead can choose to display however they are comfortable or think will show off their FE skills, then it's clear that the display side is the extra requirement.
> > --@sojeri

> I'm torn on this - I like providing examples/guidance in general, but I also want to avoid being too leading. Providing a screenshot might reveal too much information on how we'd expect them to answer. I don't know 😕
> --@suzubara 

> In the past when it was suggested that we provide an example of the expected output, @ferlatte demurred because part of the exercise is to evaluate how a candidate thinks through details where client requirements don't provide clear guidance.
> --@rpdelaney 

gonna bring this all together in a code suggestion but wanted to have your comments here for context :)

Reply:
```suggestion
warning to `stderr` and drop the row from your output.

### For browser (JSON) solutions only

Since you don't have to deal with unicode edge cases, you may have some extra time within the 4 hour limit to work on the display. Please get as far as you can within the time limit without going over -- even if it means sacrificing on style.
```

^^ what about something like this to suggest working on the display if folks have extra time?

Reply:
Why don't they have to deal with Unicode edge cases? Are we assuming that the API is perfect? That's bad. Are we assuming the browser won't generate an error? Are you sure?

Reply:
@ferlatte It's more that if we don't remove something, we're asking candidates who want to run a solution in the browser to now handle the additional requirement of making it generate valid markup. Where does that time come from?